qa/L0_sequence_batcher/test.sh

```diff
@@ -200,27 +205,36 @@ for MODEL in $MODELS; do
       (cd queue_delay_models/$(basename $MODEL)_full && \
         sed -i "s/$(basename $MODEL)/$(basename $MODEL)_full/" config.pbtxt && \
         sed -i "s/minimum_slot_utilization: 0/minimum_slot_utilization: 1/" config.pbtxt)
+  else
+    cp -r $MODEL queue_delay_models/$(basename $MODEL)_full && \
```

Comment:
Why wasn't this needed before?

Reply:
When testing for valgrind, the QUEUE_DELAY_TESTS is set to test_queue_delay_full_min_util which does indeed need the _full models.. I am not sure why this test was passing before but it should not have. 
@aramesh7 you added the valgrind test coverage that required the _full models [here](https://github.com/triton-inference-server/server/commit/5feabbde052e61cab7c822a53fbbb58effa7a622#diff-13d0a7efc37da7e6d411c65de0b4b4d91ed150a97ef5e885d92395f6506177fb).
Can you confirm why the _full models were not being created for the valgrind variant even though they appear to be needed by the test_queue_delay_full_min_util test?